modules/tinymce/src/themes/silver/test/ts/browser/editor/header/StickyHeaderInitialPlacementTest.ts

```diff
@@ -17,6 +17,7 @@ describe('browser.tinymce.themes.silver.editor.header.StickyHeaderInitialPlaceme
     it(`Test toolbar initial placement with toolbar_location: ${test.location} and height: ${test.height}`, async () => {
       const editor = await McEditor.pFromSettings<Editor>({
         base_url: '/project/tinymce/js/tinymce',
+        promotion: false,
```

Comment:
We probably should put this in the mcagar overrides instead maybe, as we wouldn't want this showing in most cases. See https://github.com/tinymce/tinymce/blob/6afd94ff80c9ffdebae118136f069a2ce1ace615/modules/mcagar/src/main/ts/ephox/mcagar/api/McEditor.ts#L13 and https://github.com/tinymce/tinymce/blob/6afd94ff80c9ffdebae118136f069a2ce1ace615/modules/mcagar/src/main/ts/ephox/mcagar/loader/Loader.ts#L91